File: sdks/go/pkg/models/shared/emailalias.go
package shared

type EmailAliasTypeEnum string

const (
	EmailAliasTypeEnumEmailAlias EmailAliasTypeEnum = "email_alias"
)

// EmailAlias
// An email alias for a user.
type EmailAlias struct {
	Email       *string             `json:"email,omitempty"`
	ID          *string             `json:"id,omitempty"`
	IsConfirmed *bool               `json:"is_confirmed,omitempty"`
	Type        *EmailAliasTypeEnum `json:"type,omitempty"`
}

File: java-sdk/lib/src/main/java/models/shared/EmailAlias.java
package .models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * EmailAlias
 * An email alias for a user.
**/
public class EmailAlias {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("email")
    public String email;
    public EmailAlias withEmail(String email) {
        this.email = email;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("id")
    public String id;
    public EmailAlias withId(String id) {
        this.id = id;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("is_confirmed")
    public Boolean isConfirmed;
    public EmailAlias withIsConfirmed(Boolean isConfirmed) {
        this.isConfirmed = isConfirmed;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("type")
    public EmailAliasTypeEnum type;
    public EmailAlias withType(EmailAliasTypeEnum type) {
        this.type = type;
        return this;
    }
}